GVFS/GVFS.Upgrader/WindowsUpgradeOrchestrator.cs

```diff
@@ -0,0 +1,55 @@
+﻿using GVFS.Common;
+using GVFS.Common.FileSystem;
+using GVFS.Common.Tracing;
+using System.IO;
+
+namespace GVFS.Upgrader
+{
+    public class WindowsUpgradeOrchestrator : UpgradeOrchestrator
+    {
+        public WindowsUpgradeOrchestrator(
+            ProductUpgrader upgrader,
+            ITracer tracer,
+            PhysicalFileSystem fileSystem,
+            InstallerPreRunChecker preRunChecker,
+            TextReader input,
+            TextWriter output)
+            : base(upgrader, tracer, fileSystem, preRunChecker, input, output)
+        {
+        }
+
+        public WindowsUpgradeOrchestrator(UpgradeOptions options)
+            : base(options)
+        {
+        }
+
+        protected override bool TryMountRepositories(out string consoleError)

```

Comment:
Should this be in the base (as `virtual`) and then the Mac version can leave it unimplemented until it is supported?